src/Aspire.Dashboard/Extensions/ClaimsIdentityExtensions.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Claims;
+
+namespace Aspire.Dashboard.Extensions;
+
+internal static class ClaimsIdentityExtensions
+{
+    /// <summary>
+    /// Searches the claims in the <see cref="ClaimsIdentity.Claims"/> for each of the claim types in <paramref name="claimTypes" />
+    /// in the order presented and returns the first one that it finds.
+    /// </summary>
+    public static string? FindFirst(this ClaimsIdentity identity, string[] claimTypes)
+    {
+        if (identity is null)
```

Comment:
`identity` is not annotated as accepting `null`.

Reply:
An earlier revision accepted null and I forgot to clean that up when I changed it. Thanks.